src/interactive/Dropdown.jsx

```diff
@@ -31,6 +30,7 @@ class Dropdown extends React.PureComponent {
 	}
 
 	onClick(e) {
```

Comment:
I'd recommend doing this like we do for `<TogglePill>` so we can still let engineers specify their own stuff to happen `onClick`:
````
	onClick(e) {
        e.preventDefault();
		this.props.toggleContent();

		if (this.props.onClick) {
			this.props.onClick(e);
		}
	}
````

Reply:
Then just do this after we set up props at the beginning of the `render()` method:
`delete other.onClick;`

This way, a new `onClick` prop isn't passed into the element, but whatever the engineer said should happen `onClick` will happen in addition to `this.toggleContent`